tensorflow_serving/sources/storage_path/file_system_storage_path_source.cc

```diff
@@ -76,6 +76,84 @@ std::set<string> GetDeletedServables(
   return deleted_servables;
 }
 
+//Add a new ServableData for the model version to the the vector of versions to aspire
+void AspireVersion(
+    const FileSystemStoragePathSourceConfig::ServableToMonitor& servable,
+    const string& version_relative_path,
+    const int version_number,
+    std::vector<ServableData<StoragePath>>* versions) {
+    const ServableId servable_id = {servable.servable_name(), version_number};
+    const string full_path =
+        io::JoinPath(servable.base_path(), version_relative_path);
+    versions->emplace_back(ServableData<StoragePath>(servable_id, full_path));
+}
+
+// Attempts to parse the version path as integer, upon success writes the result to 'version_number' and returns true.
+// Otherwise returns false when fail to parse the version path.
+bool ParseVersionNumber(const string& version_path, int64 *version_number) {
+  return strings::safe_strto64(version_path.c_str(), version_number);
+}
+
+// Update the servable data to include
+// all the model versions found in the base path as aspired versions.
+// The argument 'children' represents a list of base-path children from the file system.
+// Returns true if valid servable path is found, otherwise returns false.
```

Comment:
Please change to "Returns true if one or more valid servable version paths are found, otherwise returns false."